### PR TITLE
Add missing `CNAME` DNS record creation instruction

### LINKED ISSUE #32932: Add missing `CNAME` DNS record creation instruction
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/.github/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site

### What part(s) of the article would you like to see updated?

https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain-and-the-www-subdomain-variant

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by revolter**
### Why:

In order to not receive DNS errors, even if you want the `www` subdomain to redirect to the apex domain, you still need to create the `CNAME` DNS record.

Closes: #32932

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Comment by nguyenalex836** [comment]
@revolter Thanks so much for opening a PR! I'll get this triaged for review ✨

File: content/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site.md [hunk 1]
```diff
@@ -105,6 +105,8 @@ To set up an apex domain, such as `example.com`, you must configure a custom dom
 
 {% data reusables.pages.www-and-apex-domain-recommendation %} For more information, see "[Configuring a subdomain](#configuring-a-subdomain)."
 
+1. Navigate to your DNS provider and create a `CNAME` record that points the `www` subdomain to the default domain for your site. For example, if you want to use the subdomain `www.example.com` for your user site, create a `CNAME` record that points `www.example.com` to `<user>.github.io`. If you want to use the subdomain `www.example.com` for your organization site, create a `CNAME` record that points `www.example.com` to `<organization>.github.io`. The `CNAME` record should always point to `<user>.github.io` or `<organization>.github.io`, excluding the repository name. {% data reusables.pages.contact-dns-provider %} {% data reusables.pages.default-domain-information %}
```
Comment (sophietheking) [inline content/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site.md:108]:
```suggestion
Navigate to your DNS provider and create a `CNAME` record for the `www` subdomain that points to your {% data variables.product.prodname_pages %} default domain. For example, if your site is located at `<user>.github.io`, you should create a `CNAME` record that points `www.example.com` to `<user>.github.io` Similarly, for an organization site located at `<organization>.github.io`, you should create a `CNAME` record that points `www.example.com` to `<organization>.github.io`. Ensure that the `CNAME` record points directly to `<user>.github.io` or `<organization>.github.io` without including the repository name.

{% data reusables.pages.contact-dns-provider %} {% data reusables.pages.default-domain-information %}
```

Slightly modified for better readability. ⚡  Also, it doesn't need to be a numbered list.

**Review by sophietheking — DISMISSED** [review]
@revolter Thanks for opening this PR and contributing to our Docs, we really appreciate it. 💖 I slightly modified the paragraph you added for better readability. I'll go ahead and get this merged.

Thanks again for your contribution!

**Review by sophietheking — APPROVED** [review]

File: content/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site.md (hunk 1 above)
Comment (revolter) [reply content/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site.md]: FYI, I didn't write that paragraph down, I just copied and only slightly modified it from here:

https://github.com/revolter/docs-1/blob/1801b6f218356fda2195a3b43c26f511adcd871a/content/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site.md?plain=1#L125

**Pull request merged**